azure-spring-boot/pom.xml

```diff
@@ -38,6 +39,7 @@
         <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
+            <version>20171018</version>
```

Comment:
We've removed dependency management for json. Rather than keeping using it, perhaps you could move to something else (i.e. Jackson)